fbrp/src/fbrp/runtime/docker.py

```diff
@@ -100,25 +101,26 @@ async def log_pipe(logger, pipe):
             log_pipe(util.stderr_logger(), self.proc.log(stderr=True, follow=True)),
             self.log_psutil(),
             self.death_handler(),
-            self.command_handler(),
+            self.down_watcher(self.handle_down),
```

Comment:
To be safe, should we set state to `STARTED` after we have the `proc_pid` ?

Reply:
Done